ex6/index.js

```diff
@@ -0,0 +1,37 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert side A: ");
+const sideA = prompt(">");
+
+console.log("Insert side B: ");
+const sideB = prompt(">");
+
+console.log("Insert side C: ");
+const sideC = prompt(">");
+
+ function verify() {
+    const isSideAValid = sideA < sideB + sideC;
+    const isSideBValid = sideB < sideA + sideC;
+    const isSideCValid = sideC < sideA + sideB;
```

Comment:
The values returned by `prompt()` are `strings`. You should convert these values to numbers to perform mathematical operations.